collect_app/src/main/res/values/permission_dialog_theme.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="PermissionAlertDialogTheme" parent="Theme.AppCompat.Light.Dialog.Alert">
```

Comment:
`PermissionAlertDialogTheme` should probably be renamed as other theme names have been.

Reply:
Done!